app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt

```diff
@@ -493,9 +494,17 @@ open class FullScreenPlayer : AbstractPlayerFragment() {
                 player.seekTime(1L)
             }
             resetBtt.setOnClickListener {
-                subtitleDelay = 0
-                dialog.dismissSafe(activity)
-                player.seekTime(1L)
+                val resetSubsBuilder: AlertDialog.Builder = AlertDialog.Builder(ctx)
+                resetSubsBuilder.setTitle(R.string.confirm_reset_subs_dialog)
+                resetSubsBuilder.apply {
```

Comment:
I am not a big fan of confirmation button for such a minor thing.

Reply:
I thought you asked for that change in other PR, I think I got you wrong
Will revert it

Reply:
I meant for it on the `Reset video` button you added, and not `Reset subtitles`, but that was not explain that properly.

Reply:
Oh, you think we will need this really?